omero/sysadmins/repository-move.txt

```diff
@@ -117,3 +117,41 @@ to ``/Volumes/imports/ManagedRepository``:
     If :property:`omero.managed.dir` is set then the Managed Repository and
     the OMERO data directory should be treated independently and thus be moved
     separately if necessary.
+
+Extending the Managed Repository
+--------------------------------
+
+It is possible to leave the Managed Repository in place yet have newly
+imported image files stored on a different underlying storage volume.
+For example, if your :property:`omero.managed.dir` is set to
+:file:`/mnt/omero/ManagedRepository` and, as that volume fills, it would
```

Comment:
Really a very long sentence. I would try to split it into two separate sentences.